0.10/.sti/bin/assemble

```diff
@@ -1,11 +1,13 @@
-#!/bin/bash -e
+#!/bin/bash
 
 # Prevent running assemble in builders different than official STI image.
 # The official nodejs:0.10-onbuild already run npm install and use different
 # application folder.
 [ -d "/usr/src/app" ] && exit 0
 
-source "../lib/nodejs_context"
+source scl_source enable nodejs010
```

Comment:
Not true, I'm not sourcing it anywhere in [sti-python](https://github.com/openshift/sti-python), `.bashrc` is loaded upon all `bash` invocations and that's what STI is doing.